### PR TITLE
Embed fixes and refactoring

### PR DESCRIPTION
**Pull request by royendo**

### CODE REVIEW

**Comment by royendo** [comment]
not sure why custom embed is not working, nothing changed yet, asked in slack.

Other than that, can review.

**Comment by royendo** [comment]
Before merging, need to merge https://github.com/rilldata/rill-examples/pull/7
as some changes to openrtb should fix some of the issues....

File: src/pages/api/view-iframe.js [hunk 1]
```diff
@@ -1,19 +1,19 @@
 // Get the secret Rill service token from an environment variable.
-const rillServiceToken = "rill_svc_44owbJIWSjaPYCSZFdsLz408c7wqpT7Yougv0T8hZDj53hiG1aD3Da"
+const rillServiceToken = process.env.RILL_SERVICE_TOKEN_2;
```
Comment (ericpgreen2) [inline src/pages/api/view-iframe.js:2]: A more descriptive name than `..._2` would help clarify why we need two

**Review by ericpgreen2 — APPROVED** [review]

File: src/pages/api/view-iframe.js (hunk 1 above)
Comment (royendo) [reply src/pages/api/view-iframe.js:2]: oops this is wrong, i believe i fixed it but might be in the next PR

**Pull request merged**